ocean_lib/common/ddo/status_helper.py

```diff
@@ -20,7 +20,7 @@ def enable_flag(ddo, flag_name):
         metadata_service.attributes["status"][flag_name] = True
 
 
-def disable_flag(ddo, flag_name):
+def disable_flag(ddo: object, flag_name: str) -> None:
```

Comment:
Change `object` to `DDO`

Reply:
that breaks the import logic :( It was originally like that, but it doesn't work.

Reply:
A circular import is indicative of problems with the design.  I think the easiest solution is to simply move these to `ddo.py` and make them `DDO` methods - thus replacing the `ddo` parameter with `self`.

Reply:
I would suggest adding these in a separate PR. The ddo file is bloated (partly because of all the `@property` stuff, but not only). Do you agree with adding `Any` for now and creating an issue for moving status_helper into ddo.py?

Reply:
Yes that sounds fine.

Reply:
I ended up moving them... Let's see if we can resolve this after the tests run :)